protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -800,6 +800,23 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Referral program"};
   }
 
+  // Get current volume discount program
+  //
+  // Get the on-going volume discount program.
+  rpc GetCurrentVolumeDiscountProgram(GetCurrentVolumeDiscountProgramRequest) returns (GetCurrentVolumeDiscountProgramResponse) {
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Volume discount program"};
+  }
+
+  // Get the volume discount statistics for a given epoch for all parties
+  rpc GetVolumeDiscountStatsForEpoch(GetVolumeDiscountStatsForEpochRequest) returns (GetVolumeDiscountStatsForEpochResponse) {
```

Comment:
Both the stats ones are close enough in category that they can just be tagged as `Volume discount program` aswell. The affect is has is how the API are grouped on the REST API docs and I think these three should be grouped together.